### ISSUE TITLE
ClouDNS cluster.ws has already been added

### ISSUE THREAD

**Issue by py660** [Olivr/free-domain#631]
It seems as though someone already registered cluuster.ws as a domain, not a subdomain of cluster.ws. Anyone else having the same problem?

**Comment by Ferrarisrex** [comment]
this problem is still a problem
![image](https://github.com/Olivr/free-domain/assets/91031616/10afadef-6cc0-4125-accd-31bca3bb09e0)

**Comment by py660** [comment]
# A little distraction
A problem is still a problem if it is _(by definition)_ a problem, but it doesn't stay as a problem when such a problem becomes a github issue. Then, it is an issue, not a problem.

**Comment by Ferrarisrex** [comment]
yes, however if by definition a problem is still a problem, therefore it is still a problem, then by logic, if a problem is still a problem, but then it becomes a not problem, then it is not a problem, however, this problem has not become a not problem therefore this problem is still a problem, which is a problem.
